### PR TITLE
Add first post on Flipper

### PR DESCRIPTION
**Pull request by MichaelEvans**

### CODE REVIEW

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+layout: post
+title: "Improving App Debugging with Flipper"
+date: 2020-03-10 20:10:22 -0400
+comments: true
+categories: [Android]
+description: "Improving App Debugging with Flipper"
+keywords: "Android"
+---
+
+Some time last year, Facebook released a new mobile debugging tool, named [Flipper](https://fbflipper.com/). It's essentially the successor to the widely popular [Stetho](http://facebook.github.io/stetho/), although after talking to many developers, it seems like this newer tool is relatively unknown.
```
Comment (zmdominguez) [inline source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: Maybe full stop before "although"

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+date: 2020-03-10 20:10:22 -0400
+comments: true
+categories: [Android]
+description: "Improving App Debugging with Flipper"
+keywords: "Android"
+---
+
+Some time last year, Facebook released a new mobile debugging tool, named [Flipper](https://fbflipper.com/). It's essentially the successor to the widely popular [Stetho](http://facebook.github.io/stetho/), although after talking to many developers, it seems like this newer tool is relatively unknown. 
+
+Like Stetho, Flipper has many built-in features - including a layout inspector, a database inspector and a network inspector. Unlike Stetho though, Flipper has a very extensible API which allows for tons of customization. Over the next few articles, we're going to take a look at Flipper and its plugins, the APIs it provides, and how we can leverage them to help us debug various parts of our app. This post will focus on getting set up with Flipper, as well as taking a look at two of it's most useful default plugins.
```
Comment (zmdominguez) [inline source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: > two of it's most useful default plugins.

its

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Getting Started
+
+Getting started with Flipper is really easy:
+
+- [download the desktop client](https://fbflipper.com/),
+- Add the dependencies in your build.gradle:
```
Comment (zmdominguez) [inline source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: maybe consistency on the capitalisation of the first letter here (download vs Add)

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+dependencies {
+  debugImplementation 'com.facebook.flipper:flipper:0.33.1'
+  debugImplementation 'com.facebook.soloader:soloader:0.8.2'
+
+  releaseImplementation 'com.facebook.flipper:flipper-noop:0.33.1'
+}
+```
+- Initialize the Flipper client when your application starts:
+```
+public class MyApplication extends Application {
```
Comment (zmdominguez) [inline source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: You don't want this in KT?

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+{% img center /images/2020/03/10/database_browser.png 800 800 %}
+
+Now we can easily inspect the contents of our tables, and even run queries on the live running application! 
+
+{% img center /images/2020/03/10/writing_queries.png 800 800 %}
+
+I've [pushed a branch](https://github.com/MichaelEvans/architecture-components-samples/commit/274ea5702cb5f10ea13012ce6d9fc6b6896f471e) of the Github Browser Architecture Component sample with these changes to GitHub if you'd like to try it out! Next time we'll take advantage of Flipper's extensibility to create our own plugins to make debugging our app easier!
+
```
Comment (zmdominguez) [inline source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: Might be too excited, wanna convert some of the `!` to `.`?

Comment (MichaelEvans) [reply source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: ☺️

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown (hunk 4 above)
Comment (MichaelEvans) [reply source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: 😨

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown (hunk 3 above)
Comment (MichaelEvans) [reply source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: 🔠

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown (hunk 2 above)
Comment (MichaelEvans) [reply source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: 👍

File: source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown (hunk 1 above)
Comment (MichaelEvans) [reply source/_posts/2020-03-10-improving-app-debugging-with-flipper.markdown]: ⏹

**Pull request merged**